### PR TITLE
Updates to support installation on Amazon Linux

### PR DESCRIPTION
**Pull request by philipcristiano**
Also potentially useful for other Redhat based distros.

### CODE REVIEW

**Comment by philipcristiano** [comment]
@reactiveops/technical-team-members

**Comment by philipcristiano** [comment]
@reactiveops/technical-team-members

**Comment by jmound** [comment]
@philipcristiano I didn't get a chance to run a test, but everything is in the LGTM category. With one exception- we had a stylistic rule for the ansible roles that all variables local to the role would have a common prefix, in essence, namespacing our variables for when they are set from outside the role. It's not crucial, but preferred.

File: defaults/main.yml [hunk 1]
```diff
@@ -43,3 +43,15 @@ rabbitmq_conf_ssl_options_keyfile             : "/etc/rabbitmq/ssl/{{ rabbitmq_s
...
+
+# rabbitmq federation configuration
+rabbitmq_federation_configuration: []
+rabbitmq_policy_configuration: []
+
+# ########
+# RedHat Packages
+# ########
+
+epel_package_rpm: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm"
```
Comment (jmound) [inline defaults/main.yml]: for example: `rabbitmq_epel_package_rpm` instead of `epel_package_rpm`

Comment (philipcristiano) [inline defaults/main.yml]: Gotcha, will update. Took this from a PR to the upstream repo to try and maintain some eventually similar code.

**Comment by philipcristiano** [comment]
@jmound PTAL, updated the variable.

**Comment by opennomad** [comment]
FWIW, looks good to me. I love the correction of the bare variables in particular.

**Pull request merged**